src/components/ui/icon-button/icon-button.test.js

```diff
@@ -1,6 +1,15 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
 import IconButton from '.';
 import { setup } from '../../../utils/state.mock';
 import MenuIcon from '../../icons/menu';
+import { act } from '@testing-library/react';
+
+// jest.mock('react', () => ({
```

Comment:
if this is not used can we remove this?

Reply:
Yes, thanks for noticing this. It's added by mistake.

Reply:
Yes, I cleared all comments and unused parts.